.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "google-cloud-testing/google-cloud-conformance-tests/conformance-tests"]
+	path = google-cloud-testing/google-cloud-conformance-tests/conformance-tests
+	url = git@github.com:googleapis/conformance-tests.git
```

Comment:
Just for my understanding, what does this do?

Reply:
This file is the definition for the git-submodule which is being used to track which revision the local conformance test resources came from. You can see an example of what it looks like and how it exists in the file tree [here](https://github.com/BenWhitehead/google-cloud-java/tree/firestore/conformance-tests-migration/step2/google-cloud-testing/google-cloud-conformance-tests).